MetaMask/metamask-extension
# feat: add icon for sei network

## **Description**

Adds an icon for the sei network

[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/25079?quickstart=1)

## **Related issues**


## **Manual testing steps**

1. Visit https://chainid.network
2. Connect wallet
3. Search 'sei'
4. Add the sei network, chain id 1329
5. Verify the network has an icon in metamask

## **Screenshots/Recordings**



### **Before**

<img width="359" alt="Screenshot 2024-06-05 at 11 36 07 AM" src="https://github.com/MetaMask/metamask-extension/assets/3500406/24ea619a-3f1d-4caf-b947-d89d75b14ff3">


<img width="277" alt="Screenshot 2024-06-05 at 11 36 11 AM" src="https://github.com/MetaMask/metamask-extension/assets/3500406/31eb3eca-f247-45c3-874d-dfe00f7d8f9d">


### **After**

<img width="358" alt="Screenshot 2024-06-05 at 11 34 19 AM" src="https://github.com/MetaMask/metamask-extension/assets/3500406/ca387ca6-8544-4f94-9c43-90b25e696c88">


<img width="333" alt="Screenshot 2024-06-05 at 11 34 25 AM" src="https://github.com/MetaMask/metamask-extension/assets/3500406/0538769a-913f-4158-baa2-ef7cfba52142">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
